component-library/src/app/test-page/test-page.component.ts

```diff
@@ -1,5 +1,7 @@
 import { Component, OnInit } from '@angular/core';
-
+import { FormGroup } from '@angular/forms';
+import { IJLInputComponentConfig, InputTypes } from 'ircc-ds-angular-component-library';
+// import { IJLInputComponentConfig } from 'component-lib/src/public-api';
```

Comment:
sorry - supper nit-picky but can we remove this comment pls

Reply:
i meant to remove it, my bad. will remove it.